website/events/static/events/js/main.js

```diff
@@ -124,9 +124,15 @@ document.addEventListener('DOMContentLoaded', function () {
                 window.open(event.url, '_blank');
             }
         },
-        eventRender: function ({ el, event }) {
+        eventRender: function ({ el, event, isStart, view }) {
             el.setAttribute(
                 'title', event.extendedProps.description);
+            if (isStart && view.type !== 'list') {
+                const regInfoElem = document.createElement('div');
+                regInfoElem.classList.add('mt-2');
+                regInfoElem.textContent = event.extendedProps.registration_info;
```

Comment:
```suggestion
                regInfoElem.title = event.extendedProps.registration_info;
                regInfoElem.textContent = event.extendedProps.registration_info;
```

![image](https://user-images.githubusercontent.com/1799914/132876019-51684f4d-e3c5-4031-a317-1c77f25341d9.png)